modules/Bio/EnsEMBL/Compara/PipeConfig/Example/TFProteinTrees_conf.pm

```diff
@@ -104,7 +104,7 @@ sub default_options {
 
     # species tree reconciliation
         # you can define your own species_tree for 'treebest'. It can contain multifurcations
-        'species_tree_input_file'   => $self->check_file_in_ensembl('ensembl-compara/scripts/pipeline/species_tree.treefam.topology.nw'),
+        'species_tree_input_file'   => $self->check_file_in_ensembl('ensembl-compara/conf/treefam/species_tree.topology.nw'),
```

Comment:
This file doesn't seem to exist?

Reply:
Correct. We've removed it a while back.

We could probably remove this example PipeConfig too. Last time it was used was about 5 years ago. The options aren't terribly different from our standard pipeline anyway.

Reply:
I'll delete this PipeConfig